packages/web-forms/src/components/controls/Input/InputDate.vue

```diff
@@ -0,0 +1,49 @@
+<script setup lang="ts">
+import { computed } from 'vue';
+import DatePicker from 'primevue/datepicker';
+import type { DateInputNode } from '@getodk/xforms-engine';
+import { ISO_DATE_LIKE_PATTERN } from '@getodk/common/constants/datetime.ts';
+
+interface InputDateProps {
+	readonly question: DateInputNode;
+}
+
+const props = defineProps<InputDateProps>();
+
+const value = computed({
+	get: () => {
+		if (props.question.currentState.value == null) {
+			return null;
+		}
+
+		const temporalValue = props.question.currentState.value.toString();
+		if (!ISO_DATE_LIKE_PATTERN.test(temporalValue)) {
+			return null;
+		}
+
+		// Convert to ISO string (yyyy-mm-dd) and append time for start of day local
+		return new Date(temporalValue + 'T00:00:00');
```

Comment:
PrimeVue's date picker doesn't support temporal, so we create a Date object at the start of the local day